contents/docs/cdp/destinations/airtable.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Send PostHog analytics to Airtable
+templateId: template-airtable
+---
+
+import Requirements from "../_snippets/requirements.mdx"
+import FeedbackQuestions from "../_snippets/feedback-questions.mdx"
+import PostHogMaintained from "../_snippets/posthog-maintained.mdx"
+
+It’s easier than ever to export realtime PostHog data to satisfy the Airtable fanatics in your life.
+
+<Requirements />
+
+## Configuring Airtable
+
+With data pipelines enabled, let’s get Airtable connected.
+
+First, [create](https://airtable.com/create/tokens/new) a **personal access token** in your Airtable account. You’ll need to add the `data.records:write` scope so that PostHog can create new records.
+
+Then add the **base** you want to create new records in.
+
+![Airtable token configuration](https://res.cloudinary.com/dmukukwp6/image/upload/airtable_pat_7ce7c538e6.png)
+
+Next, you’ll want to visit Airtable’s [API reference](https://airtable.com/developers/web/api/introduction) to get your base ID, table name and field names.
+
+## Configuring PostHog’s Airtable destination
+
+With these details gathered:
+
+- Access token
+- Base ID
+- Table name
+- Field names
+
+we’re ready to set up the Airtable destination.
```

Comment:
It looks a bit weird to have a new line start with a lower case, lets just make it 2 sentences
```suggestion
You should have these details now:

- Access token
- Base ID
- Table name
- Field names

With them, we’re ready to set up the Airtable destination.
```